core/extension/extension_api_dump.cpp

```diff
@@ -127,8 +128,9 @@ Dictionary NativeExtensionAPIDump::generate_extension_api() {
 			{ Variant::RECT2I, 4 * sizeof(int32_t), 4 * sizeof(int32_t), 4 * sizeof(int32_t), 4 * sizeof(int32_t) },
 			{ Variant::VECTOR3, vec3_elems * sizeof(float), vec3_elems * sizeof(float), vec3_elems * sizeof(double), vec3_elems * sizeof(double) },
 			{ Variant::VECTOR3I, 3 * sizeof(int32_t), 3 * sizeof(int32_t), 3 * sizeof(int32_t), 3 * sizeof(int32_t) },
+			{ Variant::VECTOR4, vec4_elems * sizeof(float), vec4_elems * sizeof(float), vec4_elems * sizeof(double), vec4_elems * sizeof(double) },
 			{ Variant::TRANSFORM2D, 6 * sizeof(float), 6 * sizeof(float), 6 * sizeof(double), 6 * sizeof(double) },
-			{ Variant::PLANE, (vec3_elems + 1) * sizeof(float), (vec3_elems + 1) * sizeof(float), (vec3_elems + 1) * sizeof(double), (vec3_elems + 1) * sizeof(double) },
+			{ Variant::PLANE, vec4_elems * sizeof(float), vec4_elems * sizeof(float), vec4_elems * sizeof(double), vec4_elems * sizeof(double) },
```

Comment:
This change doesn't make sense semantically. A plane data structure is composed of a normal vector (for 3D, that's a Vector3), and the distance from the origin.

I think that the entire purpose of `vec3_elems` is to make the code more semantic, since Vector3 is used inside of many other structures, so this change doesn't make much sense. Also, since Vector4 isn't used inside of other structures, it probably doesn't make much sense to define `vec4_elems`, since we don't have `rect2_elems` or similar.